rero_ils/modules/documents/jsonschemas/documents/document_provision_activity-v0.0.1.json

```diff
@@ -210,7 +209,7 @@
         },
         "startDate": {
           "title": "Date 1",
-          "description": "Normalised date used for sorting and filtering options (MARC 008). A free text date (transcripted) can be added in the field \"Statements\".",
+          "description": "Normalised date used for sorting and filtering options (MARC 008). A free text date (transcripted) can be added in the field "Statements".",
```

Comment:
The string in Weblate just escapes before this, but I don't know if this is correct.

Reply:
\" means there is a double quote in the string.
Following some special characters from the specification of json: https://www.json.org/json-en.html
```
\b  Backspace (ascii code 08)
\f  Form feed (ascii code 0C)
\n  New line
\r  Carriage return
\t  Tab
\"  Double quote
\\  Backslash character
```